daemon/manager/manager.go

```diff
@@ -48,18 +48,17 @@ var log = logger.NewLoggerScope(pkgSubsys)
 func NewCommand() *cobra.Command {
 	configs := options.NewBootstrapConfigs()
 	cmd := &cobra.Command{
-		Use:          "kmesh-daemon",
-		Short:        "Start kmesh daemon",
-		SilenceUsage: true,
+		Use:   "kmesh-daemon",
+		Short: "Start kmesh daemon",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			printFlags(cmd.Flags())
 			if err := configs.ParseConfigs(); err != nil {
 				return err
 			}
 			return Execute(configs)
 		},
-		FParseErrWhitelist: cobra.FParseErrWhitelist{
-			UnknownFlags: true,
```

Comment:
why remove this? Now if a unknown flag provided, i think it will fail to start

Reply:
It will not, as examplified in the pr description:

```bash
[root@kmesh-7jz9n kmesh]# kmesh-daemon --abc
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --bpf-fs-path=\"/sys/fs/bpf\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --cgroup2-path=\"/mnt/kmesh_cgroup2\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --cni-etc-path=\"/etc/cni/net.d\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --conflist-name=\"\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --enable-bpf-log=\"false\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --enable-bypass=\"false\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --enable-mda=\"false\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --enable-secret-manager=\"false\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --help=\"false\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --mode=\"workload\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="FLAG: --plugin-cni-chained=\"true\"" subsys=manager
time="2024-09-23T02:46:14Z" level=info msg="oldGitVersion: 1563676948 newGitVersion: 1563676948" subsys=pkg/bpf
time="2024-09-23T02:46:14Z" level=info msg="kmesh start with Restart, load bpf maps and prog from last" subsys=pkg/bpf
```

Reply:
I donot quite understand, 

```
	// UnknownFlags will ignore unknown flags errors and continue parsing rest of the flags
```


We need to set it